README.md

```diff
@@ -65,37 +71,57 @@ make install
-You can use the `Recursive` option to recursively translate all the constants and inductives which are used by *ident*.
-You can use the `qualified` option to use a qualified default name for the translated constants and inductives. The default name then has the form `Module_o_Submodule_o_ident` if *ident* lies in the `Module.Submodule` namespace.
-
-- Parametricity Translation *term* [as *name*] [arity *n*].
-
-Define a new constant named *name* obtained by computing the parametricity translation of *term*.
-
-- Parametricity Module *modulepath*.
+## Usage and Commands
 
-Recursively translate everything in a module.
-
-- Realizer *constant or variable* [as *name*] [arity *n*] := *term*.
+To load the plugin and make its commands available:
+```coq
+From Param Require Import Param.
+```
 
-Declare *term* to be the translation of a constant.
-Useful to translate terms containing section variables, or axioms.
+The command scheme for named translations is:
+```
+Parametricity <ident> as <name> [arity <n>].
+```
+For example, the following command generates a translation named `my_param`
+of the constant or inductive `my_id` with arity 2 (the default):
+```coq
+Parametricity my_id as my_param.
+```
 
-Note that both translating a term or module may lead to proof obligations (for some fixpoints and opaque terms if you did not import `ProofIrrelevence`).
+The command scheme for automatically named translations is:
+```coq
+Parametricity [Recursive] <ident> [arity <n>] [qualified].
+```
+Such commands generate and name translations based on the given identifier.
+The `Recursive` option can be used to recursively translate all the constants
+and inductives which are used by the constant or inductive with the given
+identifier. The `qualified` option allows you to use a qualified default name
+for the translated constants and inductives. The default name then has the form
+`Module_o_Submodule_o_my_id` if the identifier `my_id` is declared in the
+`Module.Submodule` namespace.
+
+Instead of using identifiers, you can provide explicit terms to translate,
+according to the following command scheme:
+```coq
+Parametricity Translation <term> [as <name>] [arity <n>].
+```
+This defines a new constant containing the parametricity translation of
+the given term.
 
-- [Global | Local] Parametricity Tactic := t.
+To recursively translate everything in a module:
+```coq
+Parametricity Module <module_path>.
+```
 
-Use the tactic t to solve proof obligations generated by the `Parametricity` command.
+When translating terms containing section variables or axioms,
+it may be useful to declare a term to be the translation of a constant:
+```coq
+Realizer <constant_or_variable> [as <name>] [arity <n>] := <term>.
+```
 
+Note that translating a term or module may lead to proof obligations (for some
+fixpoints and opaque terms if you did not import `ProofIrrelevence`). You can
+declare a tactic to solve such proof obligations:
```

Comment:
```suggestion
fixpoints and opaque terms if you did not import `ProofIrrelevence`). You need to
declare a tactic to solve such proof obligations:
```
The tactic is currently the only way to solve proof obligations (I once tried to fix the obligation mechanism but this requires some work (which would maybe be better invested in a reimplementation based on coq-elpi or metacoq)).

Reply:
OK, sounds good to me. I wasn't sure if there was a default tactic or not, like `Program` has a default tactic.

Reply:
The main difference with `Program` is that the plugin doesn't properly register obligations, so when there is something left to prove, there is no other choice than resorting on the tactic (no interactive proof possibility).